game-app/map-data/src/main/java/org/triplea/map/game/notes/GameNotes.java

```diff
@@ -19,10 +19,10 @@ public class GameNotes {
   public static String loadGameNotes(final Path xmlGameFile) {
     Preconditions.checkArgument(
         Files.exists(xmlGameFile),
-        "Error, expected file did not exist: " + xmlGameFile.toAbsolutePath());
+        "File Existantial Error, expected file did not exist: " + xmlGameFile.toAbsolutePath());
     Preconditions.checkArgument(
         !Files.isDirectory(xmlGameFile),
-        "Error, expected file was not a file: " + xmlGameFile.toAbsolutePath());
+        "File Type Error, expected file was not a file: " + xmlGameFile.toAbsolutePath());
```

Comment:
I agree that this error message is not ideal, but I don't think adding "File Type" does help with that.
Instead I'd recommend using something like "Expected '<filename>' to be a file, but was a directory instead!", which is certainly better than claiming that a "file was not a file".